### DISCUSSION TITLE
【keyball61】キーボードが認識されません。

### DISCUSSION THREAD

**Discussion by ALEX-epower** [Yowkees/keyball#674] [Q&A] [answered]
添付した写真のように、PCとProMicroを接続すると赤いLEDは点灯するのですが、OLED上には何も表示されません。
はんだ付けは汚い部分はありますが、一見全て出来ているように思えます。
どこかおかしい個所や解決策等思いつきましたら、ご教授いただきたいです。
![P_20241011_215029](https://github.com/user-attachments/assets/bface668-f63b-4440-b923-c2cc126ae214)
![P_20241011_215057](https://github.com/user-attachments/assets/ad278d47-9688-4606-bb75-928c3a64fbf2)
![P_20241011_214641](https://github.com/user-attachments/assets/c860a484-6066-4293-8ca0-3bcca0bba76b)
![P_20241011_214714](https://github.com/user-attachments/assets/a3b61bb3-b20f-4777-ba28-03d5b789f95f)
![P_20241011_214835](https://github.com/user-attachments/assets/5359a982-a83b-46b2-9e14-fd5a82839cf9)
![スクリーンショット (49)](https://github.com/user-attachments/assets/601892ac-9482-4d86-bf1d-3391a8ac39ce)

**Comment by jtuyen** [comment]
Have you tried using QMK toolbox to manually flash the "Keyball61_test_v1.3.2" firmware to see if the OLEDs work?

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。回答が遅くなり申し訳ありません。

ProMicroへの書き込みが済んでいないとOLEDは点灯しませんが、ファームウェア書き込みはうまく出来ましたでしょうか？
半田付けはとってもキレイなので、ソフト的な問題のように思います。
ProMicroはUSB接続後にリセットスイッチを２回素早く押すと、書き込みモードに入って、対応デバイスに表示されると思います。

お手数ですがご確認お願いいたします。

**Reply by ALEX-epower** [reply]
期間をおいてやり直したら無事動きました！
どうやら接触不良？の類だったようです。対処方法を丁寧に教えて下さりありがとうございました。

**Comment by Yowkees** [comment]
無事に直ったようで良かったです！ぜひ末永くKeyball61をご愛用いただけましたら幸いです。
